typescript/sdk/src/core/HyperlaneCoreChecker.ts

```diff
@@ -262,6 +266,48 @@ export class HyperlaneCoreChecker extends HyperlaneAppChecker<
     }
   }
 
+  async checkDefaultIsmInterchainGasPaymaster(local: ChainName): Promise<void> {
+    const coreContracts = this.app.getContracts(local);
+    const defaultIsmIgp = coreContracts.defaultIsmInterchainGasPaymaster;
+
+    // Construct the violation, updating the actual & expected
+    // objects as violations are found.
+    // A single violation is used so that only a single `setDestinationGasOverheads`
+    // call is generated to set multiple gas overheads.
+    const gasOverheadViolation: DefaultIsmIgpDestinationGasOverheadsViolation =
+      {
+        type: CoreViolationType.DefaultIsmInterchainGasPaymaster,
+        subType: DefaultIsmIgpViolationType.DestinationGasOverheads,
+        contract: defaultIsmIgp,
+        chain: local,
+        actual: {},
+        expected: {},
+      };
+
+    const remotes = this.multiProvider.getRemoteChains(local);
```

Comment:
I think this relies on all remotes set in the MP, but what if the MP contains a chain that is not expressed in config? @jmrossy can correct me if I'm wrong, but consumers should not assume that MPs not contain "extra" chains

Reply:
In this case, i think its kind of implicit that an unknown chain will be undefined, and then BigNumber.from(undefined) is 0?

Reply:
good point, I'll just move over to `this.app.remoteChains`

Reply:
Nam is correct